### PR TITLE
Improve czech localization

### PR DESCRIPTION
**Pull request by BlueManCZ**
Reviewed and improved Czech localization. Added missing translations.

### CODE REVIEW

File: localization/languages/cs.json [hunk 1]
```diff
@@ -81,9 +81,9 @@
     "findMatchesPlural": "%i z %t výsledků",
     /* Focus mode */
     "isFocusMode": "Jste v režimu zaměření.",
-    "closeDialog": "OK", //used as a label for the button that closes the dialog
-    "focusModeExplanation1": "V režimu zaměření nemůžete přepínat záložky nebo měnit dimenze.",
-    "focusModeExplanation2": "Můžete opustit režim zaměření odškrtnutím \"režim zaměření\" v menu zobrazení.",
+    "closeDialog": "Rozumím", //used as a label for the button that closes the dialog
```
Comment (PalmerAL) [inline localization/languages/cs.json:84]: If I understand correctly, this means "I understand", which makes sense for the focus mode dialog. However, this also gets used for the [about dialog](https://github.com/minbrowser/min/blob/d8ed7acad9230e7fdba4e801560ad9b8221d66e8/main/menu.js#L409) and potentially other dialogs in the future. Does this still make sense in that case?

File: localization/languages/cs.json [hunk 2]
```diff
@@ -117,7 +117,7 @@
     "phishingErrorVisitAnyway": "Přesto stránky navštívit",
     "phishingErrorLeave": "Opustit tyto stránky",
     /* multiple instances alert */
-    "multipleInstancesErrorMessage": "Vyskytla se chyba. Prosím zavřete všechny ostatní spuštěné instance a restartujte Min.",
```
Comment (PalmerAL) [inline localization/languages/cs.json:120]: What's the reason for this change?

**Comment by PalmerAL** [comment]
Thanks! Looks good aside from comments.

File: localization/languages/cs.json (hunk 1 above)
Comment (BlueManCZ) [reply localization/languages/cs.json]: You are right. In other dialogs it doesn't make much sense. Let's use universal "close" instead.

File: localization/languages/cs.json (hunk 2 above)
Comment (BlueManCZ) [reply localization/languages/cs.json:120]: It sounds more civilized to reference Min as a browser. But since I used "browser Min" phrase in other translations, let's use it here as well.

**Comment by PalmerAL** [comment]
Looks good, thanks!
